content/02.main-text.md

```diff
@@ -317,6 +321,34 @@ While Manubot does require technical expertise, the benefits are manyfold.
 Specifically, Manubot demonstrates a system for publishing that is transparent, reproducible, immediate, permissionless, versioned, automated, collaborative, open, linked, provenanced, decentralized, hackable, interactive, annotated, and free of charge.
 These attributes empower integrating Manubot with an ecosystem of other community-driven tools to make science as open and collaborative as possible.
 
+## Future Manubot Enhancements
```

Comment:
We use sentence case for headers elsewhere.  We could day `Potential Future Manubot Enhancements` if you are concerned about being too speculative.

I'm also debating where this sub-section should go.  Having it end the Discussion is logical because it is a future outlook.  However, the `Manubot is an ideal authoring protocol` paragraph is so strong that I like closing with that.  Let's leave it here and restructure later if needed.

Reply:
What about moving it to the end of the Manubot section right before `## Authorship`?

Reply:
Yes, let's try that.  As part of our restructuring, I envision we may want to split the Manubot section into multiple subsections.  This could be one of them.